tools/packell/src/main/haskell/Options/Versioning.hs

```diff
@@ -0,0 +1,15 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
2023? 😄

Reply:
Well spotted, we'd have to update **_all_** files in the project if we change this template... Not sure if we want to do that at this point 🤷

Reply:
definitely something for a different PR